plugins/woocommerce/src/Enums/PaymentGatewayFeatures.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Enums;
+
+/**
+ * Enum class for all the payment gateway feature's values.
+ */
+final class PaymentGatewayFeatures {
```

Comment:
Noticed we miss some constants: `$gateway->supports( 'pay_button' )`, can we cover it as well, please?

Reply:
Oops, missed this one! Added in https://github.com/woocommerce/woocommerce/pull/57468/commits/683db24b3e1d2deb4245fc711a3079e798bfba88